### ISSUE TITLE
Footnote 35 (chapter 3.8)

### ISSUE THREAD

**Issue by crossxwill** [topepo/FES#87]
I think the footnote could be improved, so that non-English speakers do not believe the validation set was sampled from the training and test sets. 

> As noted in the last paragraph of Section 3.4.7, a better approach would be to allocate a separate data set from the training and test sets and to use this for discovery

Proposed revision:

> As noted in the last paragraph of Section 3.4.7, a better approach would be to allocate a data set **separate from** the training and test sets and to use this for discovery
